CMakeLists.txt

```diff
@@ -480,6 +480,13 @@ foreach (test IN ITEMS path str)
    target_link_libraries (mlib.${test}.test PRIVATE mongo::mlib)
 endforeach ()
 
+if ("cxx_relaxed_constexpr" IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+   add_executable(mlib.int128.test src/mlib/int128.test.cpp)
+   add_Test (mlib.int128 mlib.int128.test)
+   target_link_libraries (mlib.int128.test PRIVATE mongo::mlib)
```

Comment:
Suggest adding a `int128.test.c` to verify C compatibility via compilation of the header as C code:

```c
#include "./int128.h"

int
main (void)
{
   return 0;
}
```